itou/templates/dashboard/dashboard.html

```diff
@@ -443,7 +443,11 @@ <h5 class="card-header">Statistiques et pilotage</h5>
                     {% if can_view_stats_ddets %}
                         <p class="card-text">
                             {% include "includes/icon.html" with icon="activity" %}
-                            <a href="{% url 'stats:stats_ddets' %}">Voir mes statistiques privées</a>
+                            <a href="{% url 'stats:stats_ddets_overview' %}">Voir mes statistiques privées</a>
+                        </p>
+                        <p class="card-text">
+                            {% include "includes/icon.html" with icon="activity" %}
+                            <a href="{% url 'stats:stats_ddets_diagnosis_control' %}">Voir mes données 2021 du contrôle à postériori (Version beta)</a>
```

Comment:
```suggestion
                            <a href="{% url 'stats:stats_ddets_diagnosis_control' %}">Voir mes données 2021 du contrôle a posteriori (Version beta)</a>
```

A posteriori, comme a priori, sont des [locutions latines](https://bdl.oqlf.gouv.qc.ca/bdl/gabarit_bdl.asp?id=3799). Elles ne prennent donc [pas d'accent](https://www.larousse.fr/dictionnaires/francais/a_posteriori/4595). L'orthographe révisée de 1990 l'autorise mais crée une incohérence dans la langue (en plus d'un non-sens). Je propose de les enlever, on pourra toujours les remettre partout plus tard.

Reply:
Eh bien tu vas rire j'ai eu exactement la même lecture ! A ma grande surprise selon https://fr.wikipedia.org/wiki/A_priori_et_a_posteriori les accents sont possibles depuis 1990. Etant matheux puriste j'aime beaucoup les locutions latines (a fortiori hahaha) et j'étais tenté de corriger l'orthographe de Eric mais je n'ai pas osé. On est deux donc go !

TIL `nec plus ultra` est une locution latine o_O je pensais que c'était une veille expression des années 1990 lol.

Reply:
Ha ha ha !